src/core/namespace.c

```diff
@@ -1036,15 +1036,15 @@ static int make_read_only(const MountEntry *m, char **blacklist, FILE *proc_self
         if (mount_entry_read_only(m)) {
                 if (IN_SET(m->mode, EMPTY_DIR, TMPFS)) {
                         /* Make superblock readonly */
-                        if (mount(NULL, mount_entry_path(m), NULL, MS_REMOUNT | MS_RDONLY | m->flags, mount_entry_options(m)) < 0)
+                        if (mount(NULL, mount_entry_path(m), NULL, MS_REMOUNT | MS_RDONLY | m->flags, NULL) < 0)
```

Comment:
It seems that passing `NULL` is safe here. Sorry, please forget the above comment.

Reply:
It seems to work, but this commit is too sloppy.  I will squash it with the next commit.

---

(Because I think that is a better solution, than adding many words to the commit description to clarify -

> This still passes tests.  We got away with passing a NULL option string
when remounting tmpfs, because tmpfs remount only changes the passed
options.  I.e. passing NULL means "don't change anything", not "change to
the default options".  And we know for certain the filesystem is tmpfs, so
it's OK.
>
> The next commit will switch to using MS_BIND, which means the option string is *always* ignored...

)